### DISCUSSION TITLE
How to configure nested routing

### DISCUSSION THREAD

**Discussion by zhang1446802857** [posva/unplugin-vue-router#473] [Questions and Help] [answered]
How to configure nested routing

**Comment by PindaPixel** [comment, marked as answer]
The very basics can be found here https://uvr.esm.is/guide/file-based-routing.html#nested-routes
In the mentioned example, your `Users.vue` must render a `RouterView` component, which will automatically render `Users/index.vue` if it exists.

Customizing the behaviour is possible by reading the docs carefully. Generally I would advise to just stick with default behaviour to avoid unneeded complexity and added points of failure, especially since it is difficult to unit test such custom configuration.
